### PR TITLE
Chore: Warning for Support Threshold

### PR DESCRIPTION
**Pull request by RakeshUP**
Task: [APP-1434](https://aragonassociation.atlassian.net/browse/APP-1434)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1434]: https://aragonassociation.atlassian.net/browse/APP-1434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/web-app/src/containers/configureCommunity/index.tsx [hunk 1]
```diff
@@ -101,26 +101,87 @@ const ConfigureCommunity: React.FC = () => {
...
+                  <Tag label={t('labels.no')} className="ml-1.5 w-6" />
+                </div>
+              </ApprovalContainer>
+
+              {error?.message && (
+                <AlertInline label={error.message} mode="critical" />
+              )}
+              {value < 50 ? (
+                <AlertInline
+                  label="Proposals could be approved by minority rather than majority"
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/configureCommunity/index.tsx]: Translation strings 🙈

File: packages/web-app/src/containers/configureCommunity/index.tsx [hunk 2]
```diff
@@ -134,25 +195,51 @@ const ConfigureCommunity: React.FC = () => {
...
+                        >
+                          {Math.ceil(tokenTotalSupply * (value / 100)) <
+                          tokenTotalSupply
+                            ? '≥'
+                            : ''}
+                          {Math.ceil(tokenTotalSupply * (value / 100))}
+                        </p>
+
+                        <p className="flex-shrink-0 text-ui-600">
+                          of {tokenTotalSupply} Tokens
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/configureCommunity/index.tsx]: Here too hehe

File: packages/web-app/src/containers/configureCommunity/index.tsx [hunk 3]
```diff
@@ -101,26 +101,87 @@ const ConfigureCommunity: React.FC = () => {
...
+                <AlertInline label={error.message} mode="critical" />
+              )}
+              {value < 50 ? (
+                <AlertInline
+                  label="Proposals could be approved by minority rather than majority"
+                  mode="warning"
+                />
+              ) : (
+                <AlertInline
+                  label="Proposal will be approved by majority"
```
Comment (Fabricevladimir) [inline packages/web-app/src/containers/configureCommunity/index.tsx]: And here

**Review by Fabricevladimir — APPROVED** [review]
Works! Left a couple comments about translation strings

**Review by sepehr2github — APPROVED** [review]
Thanks Rakash, Seems good to go

**Pull request merged**